### PR TITLE
Add feather

### PR DESCRIPTION
**Pull request by lukakerr**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Review by commitay — APPROVED** [review]

**Comment by reitermarkus** [comment]
Thanks!

**Comment by claui** [comment]
I did a little digging; it seems like the Feather app is [the successor of Regxr](https://web.archive.org/web/20180425134004/https://hk.saowen.com/a/10aa6c97e1fc9ebc46c496b38e1c72ca66a7c4f1892e53a882409296d3ba97d8), which [appears to have been renamed to Feather](https://archive.is/gWSYL) only a few days ago. Around that time, GitHub purports to have received a [DMCA takedown notice](https://github.com/github/dmca/blob/a85d4e4ca3911b1613708df2b2eb1ca36206bd19/2018/2018-04-19-Expressions.md) from a software vendor named [Apptorium](http://apptorium.com/), who owns the copyright to the app [Expressions](http://apptorium.com/expressions) \([cask](https://github.com/caskroom/homebrew-cask/blob/648deb282f861dd84c93c4934c9f1b2ed172f264/Casks/expressions.rb)\).

The author of the DMCA claims to represent Apptorium, and goes on to allege that @lukakerr have infringed Apptorium’s copyright by publishing Regxr, which they claim to be an unauthorized copy of Expressions.

Notwithstanding those copyright allegations, I figure it’s highly unlikely for the takedown notice to be challenged, or for the Feather repo to return. I’d say we delete the cask and move on. @niklas-heer Thanks for the heads up.

**tl;dr** The app is gone, most likely for good.

**Comment by niklas-heer** [comment]
Sad to hear, but thank you very much for the research @claui!

**Comment by claui** [comment]
You’re welcome @niklas-heer! ☺️
